backend/src/v2/compiler/argocompiler/dag.go

```diff
@@ -494,3 +499,28 @@ func depends(deps []string) string {
 	}
 	return builder.String()
 }
+
+// Exit handler task happens no matter the state of the upstream tasks
+func depends_exit_handler(deps []string) string {
+	if len(deps) == 0 {
+		return ""
+	}
+	var builder strings.Builder
+	for index, dep := range deps {
+		if index > 0 {
+			builder.WriteString(" || ")
+		}
+		builder.WriteString(dep)
```

Comment:
nit: you could contract these into one line of `builder.WriteString(fmt.Sprintf(...))` or `fmt.Fprintf(&builder, ...)`

[go/gotip/episodes/29](https://goto.google.com/gotip/episodes/29)